data/hai/hai missions.txt

```diff
@@ -103,8 +121,18 @@ mission "First Contact: Unfettered"
 			`	"It means that we are unaltered, Hai as Hai were first born to be. Hai the conquerors. We were masters of a hundred worlds before human monkeys had learned the use of thumbs."`
 				goto masters
 			
+			label name
+			`	"We are Hai. Hai the conquerors. We were masters of a hundred worlds before human monkeys had learned the use of thumbs."`
+				goto masters
+			
 			label everyone
 			`	"We fight to defend ourselves from extinction," it says.`
+			`	"'Extinction?'" you ask. "Do you mean that there are no other Hai?"`
```

Comment:
While we're here, `"'Extinction?'"` should be replaced with either `"Extinction?"` (dropping the single quotes) or `"'Extinction'?"` (moving the question mark outside of the single quotes) in both this new section and where it had previously been.

Reply:
I think we should remove the single quotes.